content/blog/2020-04-27-moving-to-the-matrix/index.md

```diff
@@ -0,0 +1,163 @@
+you're registered, you will find a chat service that looks similar to our Slack
+domain. Some of the existing channels on Slack have already been re-established
+on Matrix, and some groups, like the documentation team lead by Marc Lichtman,
+have already moved their base of operations to Matrix.
+
+Other channels are still moving, and we expect the move to take a little bit.
+But technically speaking, there's nothing from stopping you to join right now!
+
+Once you've registered, you can also join our chat server from your phone using
+the Android and iOS apps provided by [Riot](https://riot.im). Other clients for
+your [desktop are also available](https://matrix.org/clients/).
+
+## Why on earth are you doing this? Isn't this a lot of work to move chat services?
+
+Yes, it is a lot of work. We have plenty of good reasons to move away from our
+current Slack setup, though:
+
+- Currently, we're on the free plan for Slack. This means that only a brief
+  history is stored for all chats. Unless someone has set up some IRC bridge,
+  and logs history themselves, it's not possible to go back in time very far to
+  read past discussions, which can be very annoying. Of course, we could start
+  paying Slack to see the full history. But...
+- Slack is fairly expensive, and is charged per user. Depending on how we count
+  users, we have between approx. 150 users (those are the active ones) and over
+  a thousand (if you count all of them). Even if we got a discounted price (us
+  being an open source project and all), we'd still be looking at some serious
+  expenditure. That by itself is fine -- we all know that good software and
+  services, even if it's free software, is not free to build and we are willing
+  to pay for it. But if we're already paying, we might as well look at
+  alternatives, right?
+- Let's say we stuck with Slack and went to a paid plan with full history. We'd
+  still be locked into the Slack ecosystem. Slack is not open source, and stores
+  all chat history on proprietary serves owned by Slack themselves. As an open
+  source project, that should at least give us a pause to think if that's really
+  what we want.
+- The Slack clients are generally fine, and have definitively improved in recent
+  months. But overall, they're nothing special, and still can become CPU hogs.
+  Very few features in the actual clients are *that* important that we felt
+  compelled to stick with Slack.
+
+After considering all of this, we realized it's time to take a step back, and
+decide objectively how we can best organize the real-time communication (aka chat)
+service for GNU Radio.
+
+## How we picked a new service
+
+There were a few criteria we specifically looked out for:
+
+- There needed to be clients for Android and iOS, so users could easily
+  participate via their phones on the go.
+- Also, a web-based client was a must for easy access on work computers etc.
```

Comment:
I'd drop "on work computers etc". A web client is a must for easy access.
Probably also more important than a mobile app.